scripts/post-install.sh

```diff
@@ -10,9 +10,9 @@ if ! id influxdb >/dev/null 2>&1; then
 fi
 chmod a+rX $BIN_DIR/influx*
 
-mkdir -p $LOG_DIR
+test -d $LOG_DIR || mkdir -p $LOG_DIR
```

Comment:
Nothing serious, but in some of my testing I saw a few `mkdir: cannot create directory` errors which I assumed was due to the directories already existing.